### DISCUSSION TITLE
How much data is too much

### DISCUSSION THREAD

**Discussion by cerspense** [ExponentialML/Text-To-Video-Finetuning#67] [General]
I'm prepping to train a general 448x256 model and have already acquired a dataset of over 7000 videos that are each 30s - 6 minutes long (before they are cut up and captioned). How much data do you think is too much to be useful for iteratively training and improving this model? I will be training on a 3090 and would like to see results within a day so I can tweak it as needed.

**Comment by ExponentialML** [comment]
Hey! The more variance in data the better, but having a quality, balanced dataset (cleaned, cropped, etc.) will always win regardless of size. You can go as high or low as you wish for whatever domain you're trying to train on.

**Reply by cerspense** [reply]
Cool! I was also wondering if n_sample_frames affects the output in any way, or if it just allows for more vram utilization?

**Reply by ExponentialML** [reply]
@cerspense Yes it does affect the output. Typically the  more frames, the more temporal information it's able to learn with the caveat of utilizing a lot of VRAM.
